ggml/src/ggml-sycl/wkv6.cpp

```diff
@@ -135,4 +136,7 @@ void ggml_sycl_op_rwkv_wkv6(ggml_backend_sycl_context& ctx, const ggml_tensor* s
                 );
             });
     });
+    // TODO: Why src0 and src1 are unused?
```

Comment:
```suggestion
```
As far as I can tell `src0` and `src1` are equivalent to `k_d` and `v_d` above, see https://github.com/ggerganov/llama.cpp/blob/master/ggml/src/ggml-sycl/ggml-sycl.cpp#L4039

This looks like a common function interface which was mostly useful for binary operators and re-used for other functions. It could be improved but no harm in keeping it that way either.